recipe/bld.bat

```diff
@@ -0,0 +1,18 @@
+setlocal EnableDelayedExpansion
+
+cmake -B build ^
+  -G "Ninja" ^
+  -DSUITESPARSE_ENABLE_PROJECTS="suitesparse_config;amd;btf;camd;ccolamd;colamd;cholmod;cxsparse;ldl;klu;umfpack;paru;rbio;spqr" ^
```

Comment:
```suggestion
  -DSUITESPARSE_ENABLE_PROJECTS="suitesparse_config;amd;btf;camd;ccolamd;colamd;cholmod;cxsparse;ldl;klu;umfpack;paru;rbio;spqr" ^
  -DBLA_VENDOR=All ^
```

Otherwise suitesparse uses the unsupported `BLA_VENDOR=Any` value.

Reply:
@minrk this is required to solve the `find_package(BLAS)` problem.